rust/src/server/invitations.rs

```diff
@@ -141,14 +141,31 @@ pub fn get_creator(
         .map_err(|err| APIJsonResponse::api_error_with_internal_error(err, &auth_info.language))
 }
 
+#[get("/public/<id>/creator")]
+pub fn get_creator_public(
+    id: String,
+    state: State<Storage>,
+) -> APIResult<JsonValue> {
+    get_connection(state)
+        .and_then(|mut conn| {
+            get_invitation_creator(&mut conn, &id).and_then(|data| {
+                Ok(Json(APIResponse {
+                    success: true,
+                    result: Some(data),
+                }))
+            })
+        })
+        .map_err(|err| APIJsonResponse::api_error_with_internal_error(err, &"en".to_string()))
```

Comment:
lang is english because it is for the website, and this service does not require authentication